tests/Makefile.am

```diff
@@ -244,5 +244,6 @@ AM_TESTS_ENVIRONMENT =								\
 	UEB_TEST_DATA_PATH=$(top_srcdir)/tests/ueb_test_data			\
 	LD_LIBRARY_PATH=$(top_builddir)/liblouis/.libs:$$LD_LIBRARY_PATH	\
 	PATH=$(top_builddir)/tools:$$PATH					\
+	WINEPATH=$(top_builddir)/tools						\
```

Comment:
What does this do?

Reply:
According to the `wine64` man page it 
> Specifies additional path(s) to be prepended to the default Windows PATH environment variable.

Without that the build wouldn't find my `lou_checkyaml`. No idea why it worked before.

Reply:
Maybe because the default Windows PATH wants `;` as a separator?